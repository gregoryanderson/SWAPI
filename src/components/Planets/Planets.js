import React from 'react';
import './Planets.css';

function Planets () {
    return (
        <h1>ALYZZA IS A DIFFERENT WAY TO SPELL ALYSSA</h1>
    )
}

export default Planets;