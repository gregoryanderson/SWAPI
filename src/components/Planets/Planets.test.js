import React from 'react';
import { shallow } from 'enzyme';
import Planets from './Planets'

describe('Planets', () => {
    it('should match the snapshot', () => {
        const wrapper = <Planets />;
        expect(wrapper).toMatchSnapshot();
    })
})