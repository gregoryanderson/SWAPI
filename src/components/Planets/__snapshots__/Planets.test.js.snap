// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Planets should match the snapshot 1`] = `<Planets />`;
